amplify/backend/function/bridgeXYZMutation/src/handlers/checkKyc.ts

```diff
@@ -0,0 +1,202 @@
+import fetch from 'cross-fetch';
+import { utils } from 'ethers';
+import { v4 as uuid } from 'uuid';
+import { getLiquidationAddresses, getExternalAccounts } from './utils';
+import { graphqlRequest } from '../utils';
+import { createLiquidationAddress, getUser } from '../graphql';
+import { AppSyncResolverEvent, HandlerContext } from '../types';
+import { Maybe } from 'graphql/jsutils/Maybe';
+import { KycStatus, User } from '../../../../../../src/graphql/generated';
```

Comment:
A MEGA relative import. Yuck. I know. But I'll probably deal with this in another PR if I really have to.

Reply:
Yeah would be great if it's possible to add an @amplify alias or something!